sp800-63d/vot_mapping.md

```diff
@@ -118,25 +118,87 @@ FAL maps into the VoT component A using a numeric identifier. Only one numeric i
 
 #### Assertion Presentation
 
-In addition, the presentation mechanism MAY be indicated using an alphabetic identifier. Multiple aphabetic identifiers MAY be used simultaneously. The presentation mechanism SHALL be sufficient or greater than that which is required by the FAL value indicated in the vector.
-
 |Presentation|Component Value|
 |:----:|:--:|
 |Front channel|Af|
 |Back channel|Ab|
 |Additional features implemented beyond requirements for asserted FAL|Ax|
 
-Any discrepancy in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
+## How to Use the Values in Vectors
 
-### Authenticator Management to VoT
+The implemented IAL, AAL, and/or FAL component value(s) SHALL be indicated in the vector. Only one numeric identifier SHALL be used for each xAL. Any discrepancies in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
 
-Authenticator lifecyle management, per 800-63B Section 6, maps into the VoT component M using an alphabetic identifier. Authenticator management method MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously.
+Identity proofing characteristics, authenticator type(s) used, authentication characteristics, authenticator lifecycle management characteristics, and assertion presentation values MAY be indicated in the vector. Per the requirements in the following sections, some of these values SHALL be indicated in the vector depending on the approaches implemented. Multiple alphabetic identifiers MAY be used simultaneously. 
 
-|Credential Management Method|Component Value|
-|:----:|:--:|
-|Credential issued/bound during proofing session|Mp|
-|Credential issued/bound after proofing session (remote)|Mr|
-|Credential issued/bound after proofing session (in-person)|Mi|
-|Second factor bound to a single-factor account|Ms|
-|Authentication factors reestablished (account recovery) using full proofing process|Ma|
-|Authentication factors reestablished (account recovery) using abbreviated proofing process|Mb|
\ No newline at end of file
+Identity proofing characteristics, authenticator type(s) used, authentication characteristics, and assertion presentation values SHOULD be indicated only when they demonstrate the implementation of approaches beyond those required by the xAL indicated in the vector. The appropriate value that signifies this (i.e., Px, Cx, and Ax) SHOULD also be indicated. Values SHOULD NOT be indicated that represent approaches already required by the xAL indicated in the vector. These approaches are assumed to be implemented per the xAL's requirements. 
```

Comment:
ok, if this is the only use case for the `x` values then yes it's useless -- I had assumed it was "there's something more that I don't have a more specific thing to talk about"

Reply:
When you say 'then yes it's useless', are you referring to all of the x values, or just Px (since you suggested leaving Px in the document for feedback in an earlier comment)? Either way, I'd be happy to leave all of the x values in and see what kind of feedback we get. Or we could just remove Px.

Reply:
Ok, you're right -- I was thinking that it was a SHOULD at AAL1, but if it's required across all AAL's then we should remove this value.

Reply:
My understanding was that `Px`, `Cx`, and `Ax` were to signal that there was something else you did that wasn't covered by another element. Basically, "we did IAL1 and also some other stuff". If you're only allowed to use it when you can also describe exactly what you did, as the current text suggests, then it's redundant and doesn't really add to the expressiveness. I'd say drop all of the `x` categories in that case, which has the nice side effect of making it simpler.

Reply:
Got it, thanks, that makes sense. @paul-grassi This is probably worth a larger discussion. Perhaps the x values are more interesting if they don't have to be asserted along with the value that demonstrates exactly what they did - but in that case, I would think they should have to explain somewhere why they think they're going beyond their asserted level. And that step would certainly add an additional level of complication.